documentation.html

```diff
@@ -439,10 +439,10 @@ <h4 class="display-inlineBlock">Adding dark-mode as a class</h4>
               </div>
             </section>
             <br><br/>
-        </section class="instruction-block" id="dark-mode">
+          <section class="instruction-block" id="dark-mode">
         <h4 class="display-inlineBlock">Adding dark-mode as an attribute</h4>
         <div class="text">
-          Adding the  <code>data-theme="dark"</code>  as an attribute to a body element or parent element.It can be added to any container element 
+          Adding the  <code>data-theme="dark"</code>  as an attribute to a body element .It can be added to any container element 
           of the button
          <br><br>
          <body data-theme="dark">
```

Comment:
Don't use body here, use a div

Reply:
div in the 448th line right? or someother line?

Reply:
yes

Reply:
when i change it to div data-theme="dark" in the 448th line, the background changes to dark, it dosent match ,so i dont know whether its correct, would like help